src/runtime/vdom/vdom-annotations.ts

```diff
@@ -35,8 +35,8 @@ export const insertVdomAnnotations = (doc: Document, staticComponents: string[])
       if (orgLocationNode != null) {
         const nodeRef = orgLocationNode['s-nr'];
 
-        let hostId = nodeRef['s-host-id'];
-        let nodeId = nodeRef['s-node-id'];
+        let hostId = nodeRef?.['s-host-id'];
```

Comment:
I wonder if we should just not enter the if block entirely if `nodeRef` is undefined, e.g.

```patch
- if (orgLocationNode != null) {
+ if (orgLocationNode != null && orgLocationNode['s-nr']) {
```

Reply:
yeah I think that makes sense to me, I'll make that change